sycl/test/esimd/simd.cpp

```diff
@@ -212,14 +220,12 @@ bool test_replicate2() __attribute__((sycl_device)) {
   simd<int, 8> v0(0, 1);
   auto v0_rep = v0.replicate<2, 4, 2>(1);
 
-  return v0_rep[0] == v0[1] && v0_rep[1] == v0[2] &&
-         v0_rep[2] == v0[5];
+  return v0_rep[0] == v0[1] && v0_rep[1] == v0[2] && v0_rep[2] == v0[5];
 }
 
 bool test_replicate3() __attribute__((sycl_device)) {
   simd<int, 8> v0(0, 1);
   auto v0_rep = v0.replicate<2, 4, 2, 2>(1);
 
-  return v0_rep[0] == v0[1] && v0_rep[1] == v0[3] &&
-         v0_rep[2] == v0[5];
+  return v0_rep[0] == v0[1] && v0_rep[1] == v0[3] && v0_rep[2] == v0[5];
```

Comment:
please add a testcase for `%=`

are you going to create executable tests for new esimd functionality you add?

Reply:
test added. Yes, I will modify some existing examples to use the new esimd functionality.

Reply:
@kychendev @kbobrovs please review and merge timely